### PR TITLE
feat: update cli docs for sln support

### PR DESCRIPTION
**Pull request by aryehbeitz**

### CODE REVIEW

**Comment by aryehbeitz** [comment]
I would appreciate comments for alternate location for documenting this

File: _docs_using-snyk/03_test.md [hunk 1]
```diff
@@ -30,6 +30,11 @@ title: Test
 <div class="highlight"><pre><code class="language-console" data-lang="console"><span class="go">$ snyk test --file=package.json</span>
 </code></pre></div>
 
+<p>For .NET and .NET Core projects, you can specify in the <code>--file</code> parameter the location of the solution file and the CLI will run on all the projects it finds inside.</p>
```
Comment (tkadlec) [inline _docs_using-snyk/03_test.md]: I'm missing context, but why are we calling this out? Is it because we don't auto-detect `.sln` files at the moment?

Comment (aryehbeitz) [reply _docs_using-snyk/03_test.md]: we're adding this because `.sln` support was added with the latest release `1.67.0`

Comment (tkadlec) [reply _docs_using-snyk/03_test.md]: Right, but does the CLI not auto-detect `.sln` files?

Comment (aryehbeitz) [reply _docs_using-snyk/03_test.md]: I understand you now.
I was asked to add this to the docs. I'm not sure where.
About auto-detecting `.sln`, I'm sure we could implement that - at the moment it does not.

**Pull request merged**